src/RPCServer.ts

```diff
@@ -231,8 +226,8 @@ class RPCServer extends EventTarget {
     }
 
     // Log and dispatch an event after the destruction has been completed
-    this.dispatchEvent(new events.EventRPCServerDestroyed());
-    this.logger.info(`Destroyed ${this.constructor.name}`);
+    this.dispatchEvent(new events.EventRPCServerStopped());
```

Comment:
Same here, we shouldn't dispatch stopped here, that's handled by the `async-init` decorated event now.